feat: add login logout proxy methods (#697)
jina-ai/docarray: docs/fundamentals/jina-ai-cloud/login.md

(login)=
# Authentication

To manage your resources in Jina AI Cloud using DocArray, you need to authenticate to Jina AI Cloud.

> Jina AI Cloud offers several ways to login. Read more about [Login & Token Management in Jina AI Cloud](https://docs.jina.ai/jina-ai-cloud/login/).

Reviewer: ```suggestion
Jina AI Cloud offers several ways to log in. Read more about [Login & Token Management in Jina AI Cloud](https://docs.jina.ai/jina-ai-cloud/login/).
```

> DocArray also offers convenience methods to login/logout using Python API.

Reviewer: ```suggestion
DocArray also offers convenience methods to log in and log out using the Python API.
```

## Login

> To login using Python API, use the method {meth}`~docarray.helper.login`:

Reviewer: ```suggestion
To log in using the Python API, use the {meth}`~docarray.helper.login` method:
```

```python
from docarray import login

login()
```

> The method {meth}`~docarray.helper.login` is interactive, meaning that it will prompt you to login in browser. Non-interactive login options are 

Reviewer: ```suggestion
The {meth}`~docarray.helper.login` method is interactive, meaning that it will prompt you to log in using a browser. Non-interactive login options are 
```

> available in [this page](https://docs.jina.ai/jina-ai-cloud/login/).

Reviewer: ```suggestion
explained in [Login & Token Management](https://docs.jina.ai/jina-ai-cloud/login/).
```

## Logout

> To logout, you can use the method {meth}`~docarray.helper.logout`:

Reviewer: ```suggestion
To log out, you can use the {meth}`~docarray.helper.logout` method:
```
